### PR TITLE
Feature/validate product price

### PR DESCRIPTION
**Pull request by FramaLuna**
## Description of Change

Added a validation to the price of the products

## Type of Change

- [ ] Bug fix (a fix to an existing functionality)
- [x] New feature (new functionality that does not break existing features)
- [ ] Breaking change (a change that may introduce issues or requires caution. Include instructions if this is marked)

## Related Issue

#3 

## Checklists

- [x] No linting errors
- [x] Relevant tests are created and run correctly

### CODE REVIEW

File: app/tests_integration.py [hunk 1]
```diff
@@ -93,3 +93,27 @@ def test_edit_user_with_valid_data(self):
...
         self.assertEqual(editedClient.email, client.email)
+
+class ProductTest(TestCase):
+    def test_zero_price_validation(self):
+        response = self.client.post(
+            reverse("products_form"),
+            data={
+                "name": "Pelota",
+                "type": "Juguete",
+                "price": str(0),
```
Comment (tomaszbk) [inline app/tests_integration.py:104]: could have used "0" instead but it's ok

File: app/tests_unit.py [hunk 2]
```diff
@@ -57,3 +58,49 @@ def test_update_client_with_error(self):
...
         self.assertEqual(client_updated.phone, "221555232")
+
+
+class ProductModelTest(TestCase):
+    def test_can_create_product(self):
+        result = Product.save_product(
+            {
+                "name": "Pelota",
+                "type": "Juguete",
+                "price": str(33),
```
Comment (tomaszbk) [inline app/tests_unit.py:69]: "33" is more pythonic but ok

File: app/tests_integration.py (hunk 1 above)
Comment (FramaLuna) [reply app/tests_integration.py:104]: porque me comentas en ingles si sabes que hablamos los dos español, queres que lo cambie?

**Review by tomaszbk — APPROVED** [review]
tests ran correctly

File: app/tests_integration.py (hunk 1 above)
Comment (tomaszbk) [reply app/tests_integration.py:104]: no es necesario pero para tener en cuenta para la proxima 👍

**Review by Salani1003 — APPROVED** [review]

**Review by MorettiFrancisco — APPROVED** [review]

**Pull request merged**